### PR TITLE
document noqa use in rules

### PR DESCRIPTION
**Pull request by stubs**
Small addition to docs.  Feel free to suggest edits to the bulk of it.

### CODE REVIEW

**Comment by NiallRees** [comment]
Thanks for this @stubs, would you be able to change in-line to inline and correct the 'surrounding' typo?

**Review by NiallRees — CHANGES_REQUESTED** [review]
As above

**Comment by pwildenhain** [comment]
Can you also add a note about how to use `--noqa` to avoid families of errors?
Ignore all parsing errors: `--noqa: PRS`
Ignore all templating errors: `--noqa: TMP`

**Comment by stubs** [comment]
👍 I'll add them tonight.

**Comment by alanmcruickshank** [comment]
> Ignore all parsing errors: `--noqa: PRS`
> Ignore all templating errors: `--noqa: TMP`

@pwildenhain - is this actually a feature right now? I just checked and I'm not sure it is. I mean I think it *should* be a feature, but we might need to set up a github issue to actually implement it first. Or am I going mad?

**Comment by stubs** [comment]
@alanmcruickshank, I can't speak for the noqa'ing templating errors, but `-- noqa: PRS` is functional. `sqlfluff lint` has responded to including the comment on problematic lines on my end, at least.

**Comment by alanmcruickshank** [comment]
> @alanmcruickshank, I can't speak for the noqa'ing templating errors, but `-- noqa: PRS` _appears_ to be functional. `sqlfluff lint` has responded to including the comment on problematic lines on my end, at least.

Awesome! Thanks for taking the time to document this. It sounds like I need to read the documentation when it's done 😁.

**Comment by pwildenhain** [comment]
Haha both have worked for me before! and if they don't work for someone in the future, hopefully they submit and issue 🤞

File: docs/source/rules.rst [hunk 1]
```diff
@@ -42,15 +42,31 @@ then it should flag on the segment FOLLOWING, the place that the desired
...
 .. code-block:: sql
 
-   a.a*a.b AS bad_3  -- noqa: L006
+   a.a*a.b AS bad_1  -- noqa: L006
 
 Multiple rules can be ignored by placing them in a comma-delimited list.
+
+.. code-block:: sql
+
+   a.a *  a.b AS bad_2,  -- noqa: L007, L006
```
Comment (pwildenhain) [inline docs/source/rules.rst:58]: Does this work even with the space? I thought only `-- noqa: L007,L006` worked but I may be wrong

Comment (stubs) [reply docs/source/rules.rst:58]: [The space should not cause an issue.](https://github.com/sqlfluff/sqlfluff/blob/master/src/sqlfluff/core/linter.py#L739)
>                 rules = [r.strip() for r in comment_remainder.split(",")]

**Review by NiallRees — APPROVED** [review]

**Comment by NiallRees** [comment]
With thanks @stubs !

**Pull request merged**